app/main.py

```diff
@@ -96,6 +96,13 @@ async def call(client_ip, req_params, required_params, optional_params=None):
         __validate_placements(other_params['placements'])
     params.update(other_params)
 
+    # validate that the version param is a valid int
+    if 'version' in params:
```

Comment:
'version' is a required param for our endpoint, but this function does take in a list as a parameter for which params are required so I kept this conditional.